infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -370,65 +370,66 @@ public static void configureBootstrapOptions(@Nonnull ServerContext context,
      */
     private static ChannelInitializer getServerChannelInitializer(@Nonnull ServerContext context,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 21 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)